app/helpers/candidates_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CandidatesHelper
```

Comment:
This isn't needed, as you are using `accepts_nested_attributes_for`, so you can remove this :-)